pkg/auth/wcp/discovery.go

```diff
@@ -26,16 +26,16 @@ func IsVSphereSupervisor(endpoint string, httpClient *http.Client) (bool, error)
 
 	resp, err := httpClient.Do(req)
 	if err != nil {
-		log.Debug("Failed to test for vSphere supervisor: %+v", err)
+		log.Infof("Failed to test for vSphere supervisor: %+v", err)
 		return false, err
 	}
 	defer resp.Body.Close()
 
 	if resp.StatusCode == http.StatusOK {
-		log.Debug("Got login banner from server")
+		log.Info("Got login banner from server")
```

Comment:
This does not look like an error path.  The old debug log will now always be printed IIUC.
Is there a way to reduce the level for this log?